README.md

```diff
@@ -51,6 +51,16 @@ Windows)*:
 $ pip install tensorflow-gpu
 ```
 
+## Upgrade Tensorflow
```

Comment:
No need to create a new header, no need to mention current version.

How about

```
To update TensorFlow to the latest version, add `--upgrade` flag to the above commands.
```

I don't really think we need to spend much time on this as people can easily get to the `--upgrade` flag of pip elsewhere. The purpose of this section is not to provide a full `pip` manual, but only to list the pip packages that TF provides.

Reply:
Done as per your request